acq4/drivers/Scientifica/scientifica.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, port=None, name=None, baudrate=None, ctrl_version=2):
                 SerialDevice.__init__(self, port=self.port, baudrate=baudrate)
                 try:
                     sci = self.send('scientifica', timeout=0.2)
-                except RuntimeError:
+                except TimeoutError:
```

Comment:
In some circumstances we might get a RuntimeError as well, so it should be

    except (RuntimeError, TimeoutError):